README.rst

```diff
@@ -203,7 +215,7 @@ protocols (version 2) will take place over several releases.
   * Existing HTTP/REST and GRPC protocols, and existing client APIs
     continue to be supported and remain the default protocols.
 
-* 20.05
+* 20.03.1
```

Comment:
Just want to note that I change the release version on the roadmap notes in the README. Please let me know if you want this reverted.